internal/testdb/ydb.go

```diff
@@ -0,0 +1,148 @@
+package testdb
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/ory/dockertest/v3"
+	"github.com/ory/dockertest/v3/docker"
+	"github.com/ydb-platform/ydb-go-sdk/v3"
+	"github.com/ydb-platform/ydb-go-sdk/v3/balancers"
+	ydbLog "github.com/ydb-platform/ydb-go-sdk/v3/log"
+	"github.com/ydb-platform/ydb-go-sdk/v3/trace"
+)
+
+const (
+	YDB_IMAGE    = "cr.yandex/yc/yandex-docker-local-ydb"
```

Comment:
Any chance we can pull this from DockerHub?

Reply:
We have an different registry `ghcr.io/ydb-platform/local-ydb` (official github registry).
Is it ok for you?
If ok I will change a registry in next commit

Reply:
Yes, that'd be great. It makes it a bit easier to maintain if the images are pulled from the same 1-2 registries. If those are down so be it.

Nothing against other registries, but if we have many different ones it increases the chances of flakes.

Reply:
Switched docker image to ghcr.io/ydb-platform/local-ydb